Lab_13_Make_Money/script.js

```diff
@@ -0,0 +1,57 @@
+// get the form from HTML
+const moneyForm = document.getElementById("money-form")
+// get coin display div from HTML
+const coinDisplay = document.getElementById("coin-display")
+
+// add event listener so DOM knows what to do at submission
+moneyForm.addEventListener("submit", e => {
+    // prevent default submission behavior
+    e.preventDefault()
+    // at submission, get values for coin type and coin count from the form
+    const coinType = document.getElementById("coin-type").value
+    const coinCount = parseInt(document.getElementById("coin-count").value)
```

Comment:
Add console logs to confirm the values are what you are expecting

```suggestion
    const coinType = document.getElementById("coin-type").value
    console.log(coinType) // undefined when any coin type is selected
    const coinCount = parseInt(document.getElementById("coin-count").value)
    console.log(coinCount) // 4 (if 4 is selected)
```

Remember to print values in the console to debug! Since the the element with id`coin-type` is returning `undefined`, that's a clue to look at the html.